src/core/manager.h

```diff
@@ -479,6 +481,10 @@ int manager_add_job_by_name(Manager *m, JobType type, const char *name, JobMode
 int manager_add_job_by_name_and_warn(Manager *m, JobType type, const char *name, JobMode mode, Set *affected_jobs,  Job **ret);
 int manager_propagate_reload(Manager *m, Unit *unit, JobMode mode, sd_bus_error *e);
 
+Manager* manager_pin_all_cgroup_bpf_programs(Manager *m);
+void manager_unpin_all_cgroup_bpf_programs(Manager *m);
+DEFINE_TRIVIAL_CLEANUP_FUNC(Manager*, manager_unpin_all_cgroup_bpf_programs);
+
```

Comment:
I'm wondering if `*_pin_*` is the best naming here. BPF has a [pinning concept ](https://lwn.net/Articles/664688/) that is different from the one used here. 

I got confused at the beginning because I thought `manager_pin_all_cgroup_bpf_programs` was pinning BPF objects to the BPF File System but it's doing a completely different thing.